platforms/windows/hax_wrapper.c

```diff
@@ -237,47 +226,40 @@ void hax_disable_irq(void)
     asm_disable_irq();
 }
 
-void hax_error(char *fmt, ...)
-{
-    va_list arglist;
-
-    va_start(arglist, fmt);
-    if (HAX_LOGE >= default_hax_log_level)
-        vDbgPrintExWithPrefix("haxm_error:", -1, 0, fmt, arglist);
-}
-
-void hax_warning(char *fmt, ...)
-{
-    va_list arglist;
-    va_start(arglist, fmt);
+static const int kLogLevel[] = {
+    DPFLTR_ERROR_LEVEL,
+    DPFLTR_TRACE_LEVEL,
+    DPFLTR_INFO_LEVEL,
+    DPFLTR_WARNING_LEVEL,
+    DPFLTR_ERROR_LEVEL,
+    DPFLTR_ERROR_LEVEL,
```

Comment:
Should the comma be removed as it is the last item?

Reply:
Done.